From 3a5848febba9ff9ff0dcc7aaa088848a76928f1b Mon Sep 17 00:00:00 2001
From: Doronin Stanislav <mugisbrows@gmail.com>
Date: Mon, 14 Mar 2022 12:26:01 +0300
Subject: [PATCH] install dest

---
 qwtconfig.pri | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/qwtconfig.pri b/qwtconfig.pri
index 99f006b..eb03793 100644
--- a/qwtconfig.pri
+++ b/qwtconfig.pri
@@ -16,18 +16,11 @@ QWT_VERSION      = $${QWT_VER_MAJ}.$${QWT_VER_MIN}.$${QWT_VER_PAT}
 # Install paths
 ######################################################################
 
-QWT_INSTALL_PREFIX = $$[QT_INSTALL_PREFIX]
-
 unix {
     QWT_INSTALL_PREFIX    = /usr/local/qwt-$$QWT_VERSION
     # QWT_INSTALL_PREFIX = /usr/local/qwt-$$QWT_VERSION-qt-$$QT_VERSION
 }
 
-win32 {
-    QWT_INSTALL_PREFIX    = C:/Qwt-$$QWT_VERSION
-    # QWT_INSTALL_PREFIX = C:/Qwt-$$QWT_VERSION-qt-$$QT_VERSION
-}
-
 QWT_INSTALL_DOCS      = $${QWT_INSTALL_PREFIX}/doc
 QWT_INSTALL_HEADERS   = $${QWT_INSTALL_PREFIX}/include
 QWT_INSTALL_LIBS      = $${QWT_INSTALL_PREFIX}/lib
-- 
2.32.0.windows.2

