From 857f6119c3ee6b5de85e1c51eb5a5297ecacae65 Mon Sep 17 00:00:00 2001
From: Doronin Stanislav <mugisbrows@gmail.com>
Date: Thu, 12 Jan 2023 15:36:10 +0300
Subject: [PATCH] remove include windows.h

---
 crypto/rand/rand.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/crypto/rand/rand.h b/crypto/rand/rand.h
index ed1f276..bd3e265 100644
--- a/crypto/rand/rand.h
+++ b/crypto/rand/rand.h
@@ -63,10 +63,6 @@
 # include <openssl/ossl_typ.h>
 # include <openssl/e_os2.h>
 
-# if defined(OPENSSL_SYS_WINDOWS)
-#  include <windows.h>
-# endif
-
 #ifdef  __cplusplus
 extern "C" {
 #endif
-- 
2.32.0.windows.2

